src/calculateRentalCost.js

```diff
@@ -1,10 +1,28 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DESCOUNT = 50;
```

Comment:
There is a typo in the variable name `LONG_TERM_DESCOUNT`. It should be `LONG_TERM_DISCOUNT`.